mbed_flasher/common.py

```diff
@@ -153,7 +155,7 @@ def _verify_mount_point(self, target, new_target):
             return -12
 
         for _ in range(MountVerifier.MOUNT_POINT_TIMEOUT):
-            mounts = os.popen('mount |grep vfat').read().splitlines()
+            mounts = check_output(["mount", "|", "grep", "vfat"], shell=True).splitlines()
```

Comment:
then, why this one add shell=True? This is only for linux usage?

Reply:
Pipe is a shell feature, it won't work without. Here input is always the same (secure) anyways.

Reply:
Could do the filtering in python however, maybe some other PR.